src/tools/commands/trigger-cron.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies.
+ */
+import { SelectControl } from '@wordpress/components';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from '../data/constants';
+
+export const TRIGGER_CRON_ACTION_NAME = 'runSelectedCronJob';
+
+const TriggerCron = ({ cronList, updateCommandParams }) => {
+	function onCronChange(selectedValue) {
+		const { hook, signature } = cronList[selectedValue];
+		updateCommandParams(TRIGGER_CRON_ACTION_NAME, { hook, signature });
+	}
+
+	function getOptions(cronList) {
+		return Object.keys(cronList).map((name) => {
+			return { label: name, value: name };
+		});
+	}
+
+	return (
+		<div className="trigger-cron-job">
+			{!cronList ? (
+				<p>Loading ...</p>
+			) : (
+				<SelectControl
+					label="Select cron job to run"
+					onChange={onCronChange}
+					labelPosition="side"
```

Comment:
This gives me a warning that React doesn't recognize it, setting the P to lowercase fixes it - `labelposition`

Reply:
Hi @louwie17. Thank you for your review.

I couldn't see the react warning on my end. I'm not sure if the prop name was wrong, since according to [the documentation](https://developer.wordpress.org/block-editor/reference-guides/components/select-control/#labelposition) the name is `labelPosition`. Just in case I tried with `labelposition` but the prop didn't work.

Reply:
Interesting, not sure why I see it, it's probably fine then.